src/contour/TerminalSession.cpp

```diff
@@ -616,6 +616,15 @@ void TerminalSession::onHighlightUpdate()
 {
     terminal_.resetHighlight();
 }
+
+void TerminalSession::playSound(terminal::Sequence::Parameters const params_)
```

Comment:
```suggestion
void TerminalSession::playSound(terminal::Sequence::Parameters const& params_)
```
Looks like it's okay to pass the parameter by cref.